docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -273,15 +273,18 @@ Linking within module documentation
 
 You can link from your module documentation to other module docs, other resources on docs.ansible.com, and resources elsewhere on the internet. The correct formats for these links are:
 
-* ``L()`` for Links with a heading. For example: ``See L(IOS Platform Options guide,../network/user_guide/platform_ios.html).``
+* ``L()`` for links with a heading. For example: ``See L(Ansible Tower,https://www.ansible.com/products/tower).`` As of Ansible 2.10, do not use ``L()`` for relative links to other Ansible documentation. Use ``R()`` instead.
```

Comment:
> As of Ansible 2.10, do not use ``L()`` for relative links to other Ansible documentation. Use ``R()`` instead.

So care needed if backporting docs fixes from devel to `stable-2.9`?

Reply:
Yes.  This is predominantly driven by collections, but it's safer to just start using R() imo.  @abadger can correct me if I'm confused on that point :-)

Reply:
Yeah, except that `R()` inside of options and returndocs won't form a link.  At the moment, the tables for options and returns are created using raw-html embedded in rst.  Because of that, sphinx doesn't get a chance to process them and thus can't turn it into a link

Reply:
`L()` and `U()` should form links in the options and return docs.  Those take a URL (relative or absolute) and therefore it has all the needed information to make a link in html without having to be processed by sphinx.  Here's an example which should show the difference between those two and `R()/M()`:

Here's what could be in the DOCUMENTATION variable:
``` yaml
options:
    - name: verify_certs
      description:
         - Verify the L(certificate, https://en.wikipedia.org/wiki/Public_key_certificate) of the server when True.
         - Use M(community.crypto.openssl_cert) to manage that on your server
```

Here's an approximation of the jinja template:
``` jinja2
.. raw:: html
   <table><tr>
  <td>{% for line in option['description'] %}
    @{line | html_ify}@
  {% endfor %}</td>
  </tr></table>
```

And here's what combining those two should generate in the rst file:
``` rst
.. raw:: html
   <table><tr>
   <td>
      Verify the <a href="https://en.wikipedia.org/wiki/Public_key_certificate">certificate</a> of the server when True.
      Use <span class="module">community.crypto.openssl_cert</span> to manage that on your server
  </td>
  </tr></table>
```

The difference between `U() / L()` and `M() / R()` is that the former have enough information to generate an `<a href=URL>` on their own.  The latter only have enough information to generate a sphinx `:ref:`.  The sphinx ref would have to be processed by sphinx (ie: outside of a `.. raw:: html` block in the rst) in order to turn into a link.